dts/bindings/led_strip/ti,tlc5971.yaml

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2021 Esco Medical ApS
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Texas Instruments TLC5971 (and compatible) LED controller
+
+  Driver bindings for daisy chains of a TLC5971 devices using a single device
+  tree node. Length of daisy chains in pixels is defined by the chain-length
+  property.
+
+  Individual global brightness control can be set in devicetree and individial
+  pixel brightness can be set using the led_strip api.
```

Comment:
The bindings language is underneath the driver layer, so we do not include software specific information in bindings descriptions when possible. Please remove this bit about the led_strip API.